hvpy/tests/test_io.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+def test_default_get_output_type():
+    params = HvpyParameters()
+    assert params.get_output_type() == OutputType.Raw
+
+
+def test_get_output_type():
+    class TestInputParameters(HvpyParameters):
+        a: int
+
+        def get_output_type(self):
+            if self.a == 1:
+                return OutputType.Raw
+            else:
+                return OutputType.String
+
+    params = {"a": 1}
+    params = TestInputParameters(**params)
+    assert params.get_output_type() == OutputType.Raw
+
+    params = {"a": 2}
+    params = TestInputParameters(**params)
+    assert params.get_output_type() == OutputType.String
```

Comment:
What value does this add?

You are defining code and testing it all in the test, this `get_output_type()` isn't real.

Reply:
yeah this test I think should just check the url parsing works. 

We should do that also for the base class to check the url is just the base url.

Reply:
Ah I see what you were trying to do.
Simply make an instance of the base class `HvpyParameters` and use that to verify `get_output_type()` is Raw and `url()` is the base url.

Reply:
That was done in the test above no?